virttest/env_process.py

```diff
@@ -363,6 +363,10 @@ def postprocess_image(test, params, image_name, vm_process_status=None):
         if params.get("backup_image", "no") == "yes":
             image.backup_image(params, base_dir, "backup", True)
 
+        if (params.get("backup_image_after_testing_pass", "no") == "yes" and
```

Comment:
`backup_image_after_testing_passed`?

Reply:
both of them are ok for me, updated, please help review, thanks!